### PR TITLE
docs(zh): translate troubleshooting

### PR DESCRIPTION
**Pull request by kecrily**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

translate troubleshooting

### CODE REVIEW

File: src/pages/zh-cn/guides/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+这可能是由于在渲染时试图访问 `window` 或 `document` 对象造成的。默认情况下，Astro 会[同构](https://en.wikipedia.org/wiki/Isomorphic_JavaScript)渲染你的组件，这意味着它在浏览器运行时间不可用的服务器上运行。你可以使用 [`client:only` 指令](/zh-cn/reference/directives-reference/#clientonly)禁用预渲染步骤。
+
+**解决方法**：尝试在渲染后访问这些对象（如 React 中的[`useEffect()`](https://reactjs.org/docs/hooks-reference.html#useeffect) 或 Vue 中的 [`onMounted()`](https://vuejs.org/api/composition-api-lifecycle.html#onmounted) 和 Svelte中的 [`onMount()`](https://svelte.dev/docs#run-time-svelte-onmount) ）。
+
+**状态**：预期 Astro 行为。
+
+#### 不是这个？
+
+**解决方案**：检查你的 [Astro](/zh-cn/corecepts/astro-components/) 或 [UI 框架](/zh-cn/corecepts/framework-components/)组件的相应文档。考虑从 [astro.new](https://astro.new) 中打开 Astro 入门模板，并在最小的 Astro 项目中对组件进行故障排除。
```
Comment (Hopelezz) [inline src/pages/zh-cn/guides/troubleshooting.md:49]: Missing `https://docs.astro.build` on both links

Suggested change:
**解决方案**：检查你的 [Astro](https://docs.astro.build/zh-cn/core-concepts/astro-components/) 或 [UI 框架](https://docs.astro.build/zh-cn/guides/troubleshooting/)组件的相应文档。考虑从 [astro.new](https://astro.new) 中打开 Astro 入门模板，并在最小的 Astro 项目中对组件进行故障排除。

Comment (delucis) [reply src/pages/zh-cn/guides/troubleshooting.md:49]: All our internal docs site links are intentionally relative — the issue flagged by our link checker here is with where the links themselves are pointing, a bit of the URL is missing:

```suggestion
**解决方案**：检查你的 [Astro](/zh-cn/core-concepts/astro-components/) 或 [UI 框架](/zh-cn/core-concepts/framework-components/)组件的相应文档。考虑从 [astro.new](https://astro.new) 中打开 Astro 入门模板，并在最小的 Astro 项目中对组件进行故障排除。
```

Comment (Hopelezz) [reply src/pages/zh-cn/guides/troubleshooting.md:49]: Ah, I see.

File: src/pages/zh-cn/guides/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+- 你在导入时是否包含了扩展名？（检查你导入的文件是否包含扩展名。例如：`.astro`、`.md`、`.jsx`、`.vue`）
+
+### 我的组件没有互动
+
+如果你的组件正在渲染（见上文），但对用户的交互没有反应，那么你可能缺少 [`client:*`指令](/zh-cn/reference/directives-reference/#client-directives) 来对组件进行激活。
+
+默认情况下，[UI 框架组件在客户端不会激活](/zh-cn/core-concepts/framework-components/#激活组件) 。如果没有提供 `client:*` 指令，它的 HTML 将直接渲染到页面上，而无需 JavaScript。
+
+::tip
```
Comment (delucis) [inline src/pages/zh-cn/guides/troubleshooting.md:83]: Fix Aside syntax — details in the WRITING guide: https://github.com/withastro/docs/blob/main/WRITING.md#custom-aside-component

```suggestion
:::tip
```

**Review by delucis — APPROVED** [review]
I cleaned up a few details, but LGTM! Thanks @kecrily 🚀

**Pull request merged**